seatunnel-connectors/seatunnel-connectors-flink/seatunnel-connector-flink-fake/src/main/java/org/apache/seatunnel/flink/fake/source/MockSchema.java

```diff
@@ -132,6 +132,7 @@ public TypeInformation<?> typeInformation() {
                 dataType = BasicTypeInfo.SHORT_TYPE_INFO;
                 break;
             case "long":
+            case "bigint":
```

Comment:
I don't think we need to add `bigint`, since we already has `BigInteger`, and I see the [Jmock doc](https://github.com/jsonzou/jmockdata#java对象), they did support `bigint`.

Reply:
i have try to using biginteger and  produces the same error. 
as a data developer, i usually used bigint in data warehouse , and it's as same as long in java .

Reply:
I think that most users of seatunnel are data developers , they are more familiar with the data types in the database and may not know java at all .

Reply:
Yes, use bigint is better, and I check the [seatunnel doc](https://seatunnel.apache.org/docs/2.1.2/connector/source/Fake#mock_data_schema-option--listcolumn_config), there are already support bigint.

You get the error is due to ST will generate BigInteger but flink doesn't support BitInteger, so you need to change the line 239.
```java
mockData = JMockData.mock(Long.class, mockConfig);
```

Reply:
it's still not work until change TypeInformation defined in line 154  `dataType = BasicTypeInfo.LONG_TYPE_INFO;`
so can we remove biginteger and set bigint as same as long ?

Reply:
I think we can, the BigInteger cannot work may due to the version of table api.